### DISCUSSION TITLE
Is it viable to separate git repos and project roots?

### DISCUSSION THREAD

**Discussion by FerroMeow** [NvChad/NvChad#2153] [Q&A] [answered]
Hello!
I love these customizations, and I want to switch away from VS Code. However, there's one thing that still stops me.
I work with a lot of wordpress plugins, and I want the ability to see the whole WordPress project, while keeping the ability to use git only in the subfolders - In a way, this is similar to a monorepo,  I guess.
So far, telescope throws errors, complaining there's no git initialized in WordPress root. Is there a way to fix that?

**Comment by siduck** [comment, marked as answer]
@konradBalcerzak  cant you just cd into them? like in command mode

**Reply by FerroMeow** [reply]
Oh, I assumed this wouldn't work, sorry.
Thank you, case closed

**Reply by siduck** [reply]
@konradBalcerzak you can try using our tabufline features for this! 

for example "abc" is your repo name and you do `cd abc` and open nvim

now abc has many subrepos and you want to work with multiple of them at the same time!  

so instead of using cd, use `tcd` command. 

tcd dir1 

then open a new tab, by `tabnew` command and your tabufline above would be clean! use tcd again for a different dir .


read & watch https://nvchad.com/docs/features#tabufline 

 In  a nutshell, its like linux/macos workspaces :D
